### DISCUSSION TITLE
Dekomposisi data time series

### DISCUSSION THREAD

**Discussion by sabrinaa311** [teamalgoritma/community#341] [Q&A] [answered]
Kak ini kira kira errornya kenapa ya
![Capstone project machine learning - RStudio 24_09_2022 00_51_39](https://user-images.githubusercontent.com/107026622/192027975-c47c14a7-edf8-4b74-908c-f8eaec55744c.png)

**Comment by fafilia** [comment, marked as answer]
Dear @sabrinaa311 ,
Error tersebut muncul karena object ts yang dibuat kemungkinan menggunakan frekuensi=1. Untuk object ts yang memiliki frekuensi=1 memang tidak bisa dilakukan decompose karena untuk decompose sendiri, setidaknya object ts harus memiliki 2 periode/cycle. Oleh karena itu, silahkan cek kembali bagaimana Anda menginisialisasi object ts "dataku_hourly".

Demikian yang bisa saya sampaikan, semoga bermanfaat.
